Userland/Libraries/LibWeb/SVG/SVGGraphicsElement.cpp

```diff
@@ -117,6 +117,9 @@ Gfx::AffineTransform SVGGraphicsElement::get_transform() const
 
 void SVGGraphicsElement::apply_presentational_hints(CSS::StyleProperties& style) const
 {
+    // FIXME: Hack to ensure unitless SVG properties (such as font-size) are parsed.
+    FIXME::TemporarilyEnableQuirksMode enable_quirks(document());
```

Comment:
No need to do it right now, just thinking out loud, but IIRC unitless lengths are a thing for all SVG attributes, right? I think a more "correct" solution would be to add a flag to the ParsingContext to say we're doing SVG shenanigans. That would also make the unitless lengths work for properties that aren't in the spec as allowing it.

Reply:
I think there may be more complications... SVG attributes are not really CSS attributes all the time (i.e. transform has a different syntax + semantics [as it modifies paths]), so to support things like that via CSS (which I think you can do), might be tricky.

Reply:
Oh hey I just found a helpful spec thing! https://svgwg.org/svg2-draft/types.html#presentation-attribute-css-value

Can't think right now exactly how that would be implemented, but at least it's specced how to handle these.